### PR TITLE
[Fix] Localization in ExposureSubmission UI Tests

### PR DESCRIPTION
**Pull request by johannesrohwer**
## Description

This small PR fixes an issue that occurs when the UI tests are executed with other languages than German.

## Checklist

* [x] Test your changes as thoroughly as possible before you commit them. Preferably, automate your test by unit/integration tests.
* [x] Set a speaking title. Format: {task_name} (closes #{issue_number}). For example: Use logger (closes # 41)
* [x] Make sure that your PR is not introducing _unnecessary_ reformatting (e.g., introduced by on-save hooks in your IDE)

### CODE REVIEW

**Review by inf2381 — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionNavigationController.swift [hunk 1]
```diff
@@ -146,6 +146,7 @@ final class ExposureSubmissionNavigationController: UINavigationController, UINa
 		barButtonItem.accessibilityIdentifier = "AppStrings.AccessibilityLabel.close"
 
 		navigationItem.rightBarButtonItem = barButtonItem
+		navigationBar.accessibilityLabel = "ExposureSubmissionNavigationContoller"
```
Comment (melloskitten) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionNavigationController.swift:149]: classic "Contoller"

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/__tests__/ExposureSubmissionUITests.swift [hunk 2]
```diff
@@ -78,7 +76,7 @@ class ExposureSubmissionUITests: XCTestCase {
 		// Open Intro screen.
 		XCTAssert(app.collectionViews.buttons["AppStrings.Home.submitCardButton"].waitForExistence(timeout: .long))
 		app.collectionViews.buttons["AppStrings.Home.submitCardButton"].tap()
-		XCTAssert(app.navigationBars["Info"].waitForExistence(timeout: .medium))
+		XCTAssert(app.navigationBars["ExposureSubmissionNavigationContoller"].waitForExistence(timeout: .medium))
```
Comment (melloskitten) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/__tests__/ExposureSubmissionUITests.swift:79]: this "Contoller" is everywhere

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/__tests__/ExposureSubmissionUITests.swift [hunk 3]
```diff
@@ -101,7 +99,7 @@ class ExposureSubmissionUITests: XCTestCase {
...
```
Comment (melloskitten) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/__tests__/ExposureSubmissionUITests.swift:102]: ufff "Contoller"

**Review by melloskitten — CHANGES_REQUESTED** [review]
Looks fine, but please spell check once again ;-)

**Comment by melloskitten** [comment]
Thanks for the change! Looks good.

**Review by melloskitten — APPROVED** [review]

**Pull request merged**